File: GrowthWare_VB/GrowthWareFramework/ModelObjects/Base/MFormatter.vb
Imports System.Reflection
Imports GrowthWare.Framework.ModelObjects.Base

Namespace ModelObjects.Base
	Public MustInherit Class MFormatter
		Inherits MProfile

		Protected m_Body As String

		Protected Overloads Sub Init(ByVal dr As DataRow)
			MyBase.Init(dr)
		End Sub
	End Class
End Namespace

File: GrowthWare_CS/GrowthWareFramework/ModelObjects/Base/MFormatter.cs
using System;
using System.Data;

namespace GrowthWare.Framework.ModelObjects.Base
{
	public abstract class MFormatter : MProfile
	{
		protected string m_Body;

		protected override void init(DataRow dr)
		{
			base.init(dr);

			base.setString(ref m_Body, ref dr, "BODY");
		}
	}
}
